Makefile

```diff
@@ -8,6 +8,16 @@ help: ## Print info about all commands
 	@echo
 	@grep -E '^[a-zA-Z0-9_-]+:.*?## .*$$' $(MAKEFILE_LIST) | awk 'BEGIN {FS = ":.*?## "}; {printf "    \033[01;32m%-20s\033[0m %s\n", $$1, $$2}'
 
+.PHONY: all
+all: 
+	go build ./cmd/gosky
+	go build ./cmd/laputa
+	go build ./cmd/bigsky
+	go build ./cmd/beemo
+	go build ./cmd/lexgen
+	go build ./cmd/stress
+	go build ./cmd/fakermaker
```

Comment:
I would just put these under the existing `make build` instead of `make all`?

Would there be any benefit to also doing `go build ./...` at the start of the list, to ensure that any hypothetical new cmds get at least built before we add them to this list? I think that would cache all the builds so it isn't any duplicate work. Probably over thinking things, might be wrong.

Reply:
That would work, though I'll probably also want to alias all->build to satisfy my muscle memory